srcpkgs/gitea/template

```diff
@@ -45,6 +45,23 @@ pre_build() {
 	cd $wrksrc
 }
 
+pre_install() {
```

Comment:
Please use the same separator uniformly throughout this entire block.  You start with `|` then switch to `:` then by the end go back to `|`.  My vote is to just use `:` uniformly.